src/components/MovieDetails.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import {
+  Modal,
+  Container,
+  Row,
+  Col,
+  Accordion,
+  Image,
+  ListGroup,
+} from "react-bootstrap";
+import { useState } from "react";
+import { useSelector } from "react-redux";
+import "../App.css";
+import Moment from "moment";
+
+const MovieDetails = (props) => {
+  const [show, setShow] = useState(true);
+  const movie = useSelector((state) =>
+    state?.movies.movies?.find((entity) => entity.id === props?.id)
+  );
+  return (
+    <div>
+      <Modal
+        show={show}
+        onHide={() => {
+          setShow(false);
+          props?.setDisplayDetails(false);
```

Comment:
Use redux action. Don't send state setters or state variables through props. Because this is your redux exercise. Follow this throughout the project.